From 61960bda23a53db8f57487814eaabf716e30dfea Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 11:37:36 -0400
Subject: [ethernet] i40e: Change variable type to avoid typecheck failure

Message-id: <1409658042-22689-96-git-send-email-sassmann@redhat.com>
Patchwork-id: 90290
O-Subject: [RHEL7.1 PATCH 095/281] i40e: Change variable type to avoid typecheck failure
Bugzilla: 1091126
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Kevin Scott <kevin.c.scott@intel.com>

Change the variable type to avoid compiler warning about int to u16
possible data truncation.

Change-ID: I5eb3b578c86513c9625ca32d2f0b57cc01d7dc98
Signed-off-by: Kevin Scott <kevin.c.scott@intel.com>
Acked-by: Greg Rose <gregory.v.rose@intel.com>
Acked-by: Shannon Nelson <shannon.nelson@intel.com>
Tested-by: Kavindya Deegala <kavindya.s.deegala@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit 9d2f98e155192c385a4f3acd1c918ff2ee37ad65)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40e/i40e_common.c b/drivers/net/ethernet/intel/i40e/i40e_common.c
index fde5aef..e19df8f 100644
--- a/drivers/net/ethernet/intel/i40e/i40e_common.c
+++ b/drivers/net/ethernet/intel/i40e/i40e_common.c
@@ -1300,7 +1300,7 @@ i40e_status i40e_aq_send_driver_version(struct i40e_hw *hw,
  struct i40e_aqc_driver_version *cmd =
   (struct i40e_aqc_driver_version *)&desc.params.raw;
  i40e_status status;
- int len;
+ u16 len;
 
  if (dv == NULL)
   return I40E_ERR_PARAM;
-- 
1.7.1